src/components/Nav.js

```diff
@@ -18,14 +20,54 @@ const Nav = () => {
       alignSelf="center"
       gap="medium"
     >
-      <RoutedAnchor
-        path="/"
-        icon={<GrommetIcon size="large" />}
-        label={size !== 'small' && <Text size="xlarge">grommet</Text>}
-      />
+      <Box
+        direction="column"
+        justify="between"
+        align="start"
+        alignSelf="center"
+        width="medium"
+      >
+        <RoutedAnchor
+          path="/"
+          icon={<GrommetIcon size="large" />}
+          label={size !== 'small' && <Text size="xlarge">grommet</Text>}
+        />
+        {themeName === 'grommet' && <Text size="small">for open source</Text>}
+        {themeName === 'hpe' && <Text size="small">for HPE</Text>}
+      </Box>
+
       <Box direction="row" gap="small">
         {!searchOpen && (
           <Box direction="row" gap="small">
+            <Button
+              plain
+              onClick={() => {
+                if (themeName === 'hpe') {
+                  setThemeName('grommet');
+                } else {
+                  setThemeName('hpe');
+                }
+              }}
+            >
+              {({ hover }) => (
+                <Box
+                  pad={{ vertical: 'small', horizontal: 'small' }}
+                  round="xlarge"
+                  background={hover ? 'active' : undefined}
+                  direction="row"
+                  gap="xsmall"
+                >
+                  <Text>
+                    {themeName === 'hpe'
+                      ? 'use for open source'
+                      : 'use for HPE'}
+                  </Text>
+                  {themeName === 'grommet' && (
+                    <HpeIcon color="#01A982" size="medium" />
```

Comment:
```suggestion
                    <HpeIcon color="plain" size="medium" />
```
This icon happens to have the color defined on the SVG, so if we apply `plain` as its color it will just use that.

Reply:
![image](https://user-images.githubusercontent.com/23144209/179321266-d629e3a9-1093-4e15-8728-75de8fd9eec2.png)
This is my result when I use plain, might there be another label?

Reply:
Hmm that's odd, plain should work. You can leave as hex code for now.

Reply:
I think before I tried using `brand` but that colored in in the Grommet purple, so I ended up just using the hex values.